Pulling in updated httpism package after security patch

hindsc52 opened the pull request:
Resolves #301 

## Summary

This PR updates the version of `httpism` which has now patched some downstream deps introducing some security issues.

Related to: ttps://github.com/featurist/httpism/pull/51

This only fixes one of 53 other security issues found when running npm audit. We should really try and keep this package as up to date as possible.

## How Has This Been Tested?

There are two unit tests failing, however these are also failing on the master branch 🤔 

![image](https://user-images.githubusercontent.com/14273378/67546391-40230200-f6f4-11e9-9a83-ae8f2edcdd91.png)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [n/a] I've added tests for my code
- [n/a] My change requires a change to the documentation.
- [n/a] I have updated the documentation accordingly.

EmmaJP commented:
Seems like a lot of changes in the package-lock, but I'm OK for this to merge if it all makes sense in relation to the needed updates. It would be good to resolve the build issues if someone can.

@andymsuk  @llouisetaylor would one of you mind verifying these changes. Thanks.

andymsuk commented:
I've talked to Chris about this outside of GitHub (sorry, probably should have done it here!)

I've run all the tests on both master and this branch, and they all pass locally for me, but @hindsc52  has said that 2 fail locally for him. I've suggested increasing the timeout on the tests.

EmmaJP commented:
Thanks @andymsuk ... if increasing the time allows it to pass, I'd be curious why the different environment is needing longer.

hindsc52 commented:
Just go around to adding a longer default timeout to Mocha, however I see . the same tests failing.

Im currently in a two day workshop, I can look at this again after unless someone is free to pick this up.

hindsc52 commented:
These test still do not run locally for me, however a number of colleagues have run the tests and the pass fine.

I did notice that travis was not running the tests at all, it failed right after the npm instal due to a missing package.

Travis was also running again node 7 where as the circle config was using node 8. I have updated travis to run against node 8 and added the missing package. test are now passing on CI

andymsuk commented:
Thanks @hindsc52. All looks good to me.

@EmmaJP - happy for me to merge and release?

hindsc52 commented:
Any chance we can get this merged please @EmmaJP

EmmaJP commented:
@andymsuk if you're happy with it I'm happy for the merge and release to go ahead.

andymsuk commented:
Thanks @hindsc52. This has now been released/published as v2.3.3.

EmmaJP commented:
Thanks @andymsuk and @hindsc52
